account_move_line_report_xls/__init__.py

```diff
@@ -21,8 +21,8 @@
 ##############################################################################
 
 try:
-    from . import account_move_line
-    from . import report
+    import account_move_line
+    import report
```

Comment:
Is there a specific reason to change that to an absolute import?
The relative import is safer and that's what should be used preferably.